GVFS/GVFS.Virtualization/Projection/GitIndexProjection.cs

```diff
@@ -1685,7 +1685,6 @@ private void AddParentFoldersToListToKeep(string parentKey, ConcurrentHashSet<st
             PlaceholderListDatabase.PlaceholderData placeholder, 
             out string gitPath)
         {
-            // TODO(Mac) This can be optimized if needed

```

Comment:
We could optimize this a bit by not calling `Replace` on Mac, but since this only gets called in rare error scenarios, seems unnecessary to worry about is perf.